src/main/java/mate/academy/springintro/rolerepository/Role.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.springintro.rolerepository;
```

Comment:
What is the reason to have a separate package for role? Is it possible to add Role to model package and RoleRepository to repository package?